components/developer/rust/patches/src_libbacktrace_configure.patch

```diff
@@ -0,0 +1,15 @@
+$NetBSD: patch-src_libbacktrace_configure,v 1.1 2017/11/16 19:48:24 jperkin Exp $
+
+Use pkgsrc libtool.
```

Comment:
Why do we need it?

Reply:
Not sure whether it changes anything for us but there were some issue related to link flags with 1.23 that I have not seen here.